internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -174,33 +174,17 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
-
-	var aliases []string
-	if s.httpsEnabled {
-		if aliases, err = convertAlias(s.manifest.RoutingRule.Alias); err != nil {
-			return "", err
-		}
-	}
-
-	aliasesFor, err := convertHostedZone(s.manifest.RoutingRule.RoutingRuleConfiguration)
-	if err != nil {
-		return "", err
-	}
 	var deregistrationDelay *int64 = aws.Int64(60)
 	if s.manifest.RoutingRule.DeregistrationDelay != nil {
 		deregistrationDelay = aws.Int64(int64(s.manifest.RoutingRule.DeregistrationDelay.Seconds()))
```

Comment:
The reason I did not move this logic inside `convertApplicationLoadBalancer` because we use this value for NLB as well, which is a bug because if we have config such as `http: false` and if customer wants to modify `deregistrationDelay` for NLB, then there is no way. This bug can be fixed separately, but time being I have kept this as is. 